python/tvm/auto_scheduler/task_scheduler.py

```diff
@@ -75,10 +76,10 @@ def make_search_policies(
         if model_type == "xgb":
             cost_model = XGBModel(num_warmup_sample=len(tasks) * num_measures_per_round)
             if load_model_file:
-                logger.info("Load pretrained model...")
+                logger.info("TaskScheduler: Load pretrained model...")
```

Comment:
You can change L37 to `logger = logging.getLogger("TaskScheduler")` to avoid this and future issues.